libs/components/src/lib/header/header.spec.ts

```diff
@@ -16,105 +18,36 @@ describe('vwc-header', () => {
 	describe('basic', () => {
 		it('should be initialized as a vwc-header', async () => {
 			expect(element).toBeInstanceOf(Header);
+			expect(element.elevationShadow).toBeFalsy();
 			expect(element.alternate).toBeFalsy();
-			expect(element.fixed).toBeFalsy();
-			expect(element.heading).toBeUndefined();
 		});
 	});
 
-	describe('fixed', () => {
-		it('should add "fixed" class to control and attribute to host when fixed is true', async () => {
-			const control = getControlElement(element);
-			const hasClassFixedOnInit = control.classList.contains('fixed');
-			element.fixed = true;
-			await elementUpdated(element);
-			expect(hasClassFixedOnInit).toEqual(false);
+	describe('elevation shadow', () => {
+		it('should default elevation to no-shadow', async () => {
+			const elevation = element.shadowRoot?.querySelector(ELEVATION_SELECTOR) as Elevation;
 
-			const hasClassFixed = control.classList.contains('fixed');
-			expect(hasClassFixed).toEqual(true);
+			expect(elevation.noShadow).toBeTruthy();
```

Comment:
Truthy can be almost everything. You expect this to equal `true`.
```suggestion
			expect(elevation.noShadow).toEqual(true);
```

Reply:
dont mind but it should only be other than `undefined`